src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionWarnOthersViewController.swift

```diff
@@ -59,17 +59,19 @@ class ExposureSubmissionWarnOthersViewController: DynamicTableViewController, EN
 		navigationFooterItem?.isPrimaryButtonLoading = true
 		navigationFooterItem?.isPrimaryButtonEnabled = false
 		exposureSubmissionService?.submitExposure { error in
-			if let error = error {
+			switch error {
+			case .noKeys:
+				self.performSegue(withIdentifier: Segue.sent, sender: self)
+			case .none:
+				self.performSegue(withIdentifier: Segue.sent, sender: self)
```

Comment:
Looks like code duplication. Could also be written in single case statement. Nothing major ... just an idea.

Reply:
Thank you, I overlooked! Will change 👍